Documentation/help_files/external_data_dashboard_help.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * External Data Dashboard Help.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @version 1.0.0
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+ 
+use OpenEMR\Core\Header;
+
+require_once("../../interface/globals.php");
+?>
+<!DOCTYPE HTML>
+<html>
+    <head>
+    <?php Header::setupHeader();?>
+    <title><?php echo xlt("External Data Help");?></title>
+    </head>
+    <body>
+        <div class="container oe-help-container">
+            <div>
+                <center><h2><a name='entire_doc'><?php echo xlt("External Data Help");?></a></h2></center>
+            </div>
+            <div class= "row">
+                <div class="col-sm-12">
+                    <p><?php echo xlt("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor");?>.
```

Comment:
Since we are a bit unclear what this thing does, prob better to not have a howto for it (especially if it's basically blank :) )

Reply:
It is hidden, will not display, saved for later